dashboard/app.py

```diff
@@ -24,11 +24,11 @@ def get_db_connection(config_file: _Environ) -> connection:
     """
     try:
         return connect(
-            database=config_file["DB_NAME"],
-            user=config_file["DB_USER"],
-            password=config_file["DB_PASSWORD"],
-            port=config_file["DB_PORT"],
-            host=config_file["DB_HOST"]
+            database=config_file["DATABASE_NAME"],
+            user=config_file["DATABASE_USERNAME"],
+            password=config_file["DATABASE_PASSWORD"],
+            port=config_file["DATABASE_PORT"],
```

Comment:
This part has reminded me, we should do a pass of all our code to standardise environment variable names before the end of the week (fine for now though)